studio/rmq_config.yml

```diff
@@ -0,0 +1,54 @@
+database: 
+    type: s3
+    endpoint: https://s3-us-west-2.amazonaws.com/
+    bucket: "karl-mutch-rmq"
+    compression: None
+    authentication: none
+
+storage:
+    type: s3
+    endpoint: https://s3-us-west-2.amazonaws.com/
+    bucket: "karl-mutch-rmq"
+    compression: None
+
+runner:
+    slack_destination: "@karl.mutch"
+
+cloud:
+    queue:
+        uri: "amqp://guest:guest@localhost:5672/%2F?connection_attempts=50"
```

Comment:
not sure I understand from the docs what's the connection between queue name and this uri, if any...

Reply:
The URI points at a RabbitMQ server that is being used by the user.  In the case of the cloud solutions the URLs are well known and hidden by APIs like boto, using the region instead to locate the s3 and sqs endpoints through well known endpoints.  In the case of RabbitMQ I took a guess as to where it could sit.  The alternative might be to add a section called 'onpremise:' but that seemed like overkill.

Reply:
After thinking on it I changed the parameter name to rmq from uri to be clear about what it is.